### DISCUSSION TITLE
Removed all video sources, how to disable "Setup Wizard" on startup?

### DISCUSSION THREAD

**Discussion by unimpededglare** [OpenSageTV/Community#28] [General]
I'm slowly phasing out SageTV and I removed all of my video capture devices from the network. I'm now just using it for media playback.

Now whenever I start SageTV, I get the "Setup Wizard" wanting me to setup a capture device. How can I disable the wizard on startup as I don't have any sources that I can use any longer?

**Comment by jusjoken** [comment]
Take a look at this [post...](https://forums.sagetv.com/forums/showthread.php?t=36134), which points to [this post which describes how to fake sagetv with an empty tuner.](https://forums.sagetv.com/forums/showthread.php?t=15871)


---
@jusjoken
If you wish to see what I am up to and support my efforts visit my [Patreon page](https://www.patreon.com/user?u=58035567)

**Reply by unimpededglare** [reply]
I tried that and I get constant delays and spinning circles in SageTV as soon as I added the fake source. I restarted the service to see if that would fix it and it doesn't. When I stop playback on a video files it gives me a spinning circle and takes several seconds to stop now.
I removed the video source (which again was constant spinning circles and delays), and now everything is working without delay again.
I'm guessing SageTV is trying to communicate with the fake network encoder and getting stuck when it doesn't get a reply back?

**Reply by jusjoken** [reply]
Sorry.  Found the post and hoped it would help.  I used to use it plus the fake encoder while coding on a PC that had no tuner... But many moons ago.

k
